TESTS/Makefile.am

```diff
@@ -25,17 +25,17 @@ bug_142_gen_SOURCES = bug_142_gen.f
 
 if ICB
 icb_arpack_c_SOURCES = icb_arpack_c.c
-icb_arpack_c_LDADD = $(top_builddir)/SRC/libarpack$(LIBSUFFIX).la
+icb_arpack_c_LDADD = $(LDADD)
 icb_arpack_c_CPPFLAGS = $(AM_CPPFLAGS) -I$(top_builddir) -I$(top_srcdir)/ICB
 
 icb_arpack_cpp_SOURCES = icb_arpack_cpp.cpp
-icb_arpack_cpp_LDADD = $(top_builddir)/SRC/libarpack$(LIBSUFFIX).la
+icb_arpack_cpp_LDADD = $(LDADD)
 icb_arpack_cpp_CPPFLAGS = $(AM_CPPFLAGS) -I$(top_builddir) -I$(top_srcdir)/ICB
 else
 bug_1315_single_SOURCES = bug_1315_single.c
 bug_1315_double_SOURCES = bug_1315_double.c
-bug_1315_single_LDADD = $(top_builddir)/SRC/libarpack$(LIBSUFFIX).la
-bug_1315_double_LDADD = $(top_builddir)/SRC/libarpack$(LIBSUFFIX).la
+bug_1315_single_LDADD = $(LDADD)
+bug_1315_double_LDADD = $(LDADD)
```

Comment:
I think these should stay the way they were, see ddfd843e29dc0a58da88ed0168e7a3c4655e7a4c.  But I'm not familiar with the problem that this tries to solve.